third-party-notices

```diff
@@ -20,3 +20,11 @@ Licensed under the MSDN Code Gallery Licenses
 
 Available at
 https://web.archive.org/web/20101223145710/http://code.msdn.microsoft.com/wictools/Project/License.aspx
+
+License notice for SixLabors.ImageSharp
+---------------------------------------
+
+Copyright (c) Six Labors and contributors.
+Licensed under the Apache License, Version 2.0.
+
+http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
@JimBobSquarePants I'm not sure how to handle the license link here with the impending change.  Would you prefer I link to the license in the repo under a current commit?

Reply:
It might be wise yeah. I'm still working out the kinks.